gui/wxpython/startup/guiutils.py

```diff
@@ -218,7 +218,7 @@ def create_location_interactively(guiparent, grassdb):
     """
     from location_wizard.wizard import LocationWizard
 
-    gWizard_output = (None, None, None)
+    gWizard_output = (None, None, "PERNAMENT")
     gWizard = LocationWizard(parent=guiparent,
                              grassdatabase=grassdb)
 
```

Comment:
You can solve the case if `gWizard.location is None` right here (and return) and continue with the "success" case below without any nesting.

Reply:
Addressed.